wordpress-mobile/WordPressKit-iOS
# Add Notification Actions

In this PR:

- Move Notification Actions to WordPressKit.
- Actions are basically empty, and delegate their actual functionality to an implementation of the `FormattableContentActionCommand` that will be provided by clients of this library (where "clients of this library" actually means any code that uses this library).

Naming is tentative, that is the best I could come up with.

At this stage, parsing Actions remains a responsibility of clients of the library. 

To test:
- Check out [this PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/9719) in the main repo
